docs/features/autoupdater.rst

```diff
@@ -3,6 +3,13 @@ Autoupdater
 
 Gluon contains an automatic update system which can be configured in the site configuration.
 
+Please mind, that asking users for consent is not implemented within the automatic update system.
```

Comment:
Please remember that...

Reply:
If we include this at all I guess we should clarify the difference between what we mean by consent vs. the already existing ability of turning the Autoupdater on or off in config mode.
Especially how consent must be informed, i.e. ensuring the user is aware of the implications of their consent.